app/mixins/desktop-only-route.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @module timed
+ * @submodule timed-mixins
+ * @public
+ */
+import Mixin from 'ember-metal/mixin'
+
+/**
+ * Mixin to prevent mobile devices to access the page
+ *
+ * @class DesktopOnlyRouteMixin
+ * @extends Ember.Mixin
+ * @public
+ */
+export default Mixin.create({
+  /**
+   * Before model hook, ensure the device is large enough. If this isn't the case redirect to index page.
+   *
+   * @method beforeModel
+   * @param {Ember.Transition} transition The transition
+   * @return {*} The return value of the parent function or a new transition
+   * @public
+   */
+  beforeModel(transition) {
+    if (
+      Object.values(
+        this.getProperties('media.isMd', 'media.isLg', 'media.isXl')
+      ).some(i => i)
```

Comment:
This looks a bit funny, but i can't think of a good alternative... What came to mind was `.some(Boolean)` or `.some(i => !!i)` but that's also not any easier on the eye.

Reply:
Well i guess most readable would be the boring way: `this.get('media.isMd') || this.get('media. ... `

Reply:
Yeah you're right..